Makefile

```diff
@@ -9,58 +9,92 @@
 # `make flowapi-down` to tear down the docker container for flowapi
 # only.
 #
- # By setting the variable FLOWDB_SERVICES you can choose which flowdb
+# By setting the variable FLOWDB_SERVICES you can choose which flowdb
 # version or versions you'd like to use when running `make up`. Examples:
 #
- #     FLOWDB_SERVICES=flowdb_testdata make up
+#     FLOWDB_SERVICES=flowdb_testdata make up
 #     FLOWDB_SERVICES=flowdb_synthetic_data make up
 #     FLOWDB_SERVICES="flowdb_testdata flowdb_synthetic_data" make up
 #
 # flowmachine and flowapi will connected to the first flowdb service in the list.
 
 DOCKER_COMPOSE_FILE_DEV ?= docker-compose-dev.yml
 FLOWDB_SERVICES ?= flowdb_testdata
-DOCKER_SERVICES ?= flowapi flowmachine redis $(FLOWDB_SERVICES)
+DOCKER_SERVICES ?= $(FLOWDB_SERVICES) flowapi flowmachine flowauth redis
 export DB_HOST=$(word 1, $(FLOWDB_SERVICES))
 
 
 all:
 
 up:
-	docker-compose -f $(DOCKER_COMPOSE_FILE_DEV) up -d $(DOCKER_SERVICES)
+	docker-compose -f $(DOCKER_COMPOSE_FILE_DEV) up -d --build $(DOCKER_SERVICES)
 
 down:
 	docker-compose -f $(DOCKER_COMPOSE_FILE_DEV) down
 
-flowdb_testdata-up:
-	docker-compose -f $(DOCKER_COMPOSE_FILE_DEV) up -d flowdb_testdata
+
+flowdb-up: flowdb-build
```

Comment:
Is there any reason we can't use pattern rules to shorten this? e.g.
```
%-up: %-build
	docker-compose -f $(DOCKER_COMPOSE_FILE_DEV) up -d --build $*
```

Reply:
Yeah, I've been going back and forth on this one.

Your solution of using patterns was my first instinct, but the reasons I went for the longer version are:
- discoverability / ease of understanding for someone not familiar with Makefile syntax
- you can't tab-complete patterns, so typing something like `make flowmachine_testdata-up` becomes tedious.

No strong opinion though, so I'm happy to switch to patterns if you think it makes it more readable.

Reply:
Good points. Yes, that makes sense.

Reply:
It's a good point, though. I've added a comment in the Makefile to explain the repetition.